src/examples/pidMotor.adoc

```diff
@@ -296,7 +296,7 @@ image:pidMotor/PIDTuned.png[Final tun of the PID controller]
 
 Go back to Examples index:
 
-link:./examplesIndex.html[Examples Index]
+link:./examplesIndex.adoc[Examples Index]
```

Comment:
Unfortunately this link did not work for me. 
Is it possible that you mean **./examples/index.adoc** ?
(full link: https://github.com/eclipse-4diac/4diac-documentation/blob/main/src/examples/index.adoc)

Reply:
thanks for noting. Yes this should be the link you are mentioning. Will fix it immediately.